docs/UserGuide.adoc

```diff
@@ -49,287 +49,289 @@ This app will not work with earlier versions of Java 8.
 +
 .  Download the latest `progresschecker.jar` link:{repoURL}/releases[here].
 .  Copy the file to the folder you want to use as the home folder for your ProgressChecker.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
++
+image::download3.png[width="790"]
++
+. After you click to open the progresschecker.jar app, you can see the following files will be automatically generated into the home folder where you copy jar file to.
++
+image::download2.png[width="790"]
++
+.  Double-click the file to start the app. The GUI like below should appear in a few seconds. +
+Now, you are ready to start to explore ProgressChecker!
 +
 image::Ui.png[width="790"]
 +
 [NOTE]
 ====
-The viewing of learning outcomes and exercises are not ready yet! Currently, only some of week 11's exercises are reflected.
+We have put some of week 11's exercises in the app. You can open the app and try it!
 ====
-+
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
-+
-[TIP]
-You can type a command and press `Tab` to auto bring out all the command parameters.
 
-* *`list`* : lists all contacts
+[[Features]]
+== Features
+
+You may find that there are lots of new commands in ProgressChecker. However, you may also find that the app is easy to use, as long as you follow the instructions below. +
```

Comment:
another possible repharsing option: "ProgressChecker provides a wide variety of user-friendly commands whose detailed usages can be found in the next section". can add hyper link to "next section" too.